src/index.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please kindly do not disable linters, it's better to fix linters it helps to write a clean and good codebase 😉